forc-plugins/forc-client/src/op/deploy.rs

```diff
@@ -56,6 +59,11 @@ pub struct DeployedContract {
     pub chunked: bool,
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, PartialOrd, Ord)]
```

Comment:
Maybe a doc comment here. Feel free to tweak this

/// Represents a deployed script on the Fuel network.
/// Scripts are deployed as blobs with generated loaders for efficiency.